tests/ppx/react/expected/topLevel.res.txt

```diff
@@ -5,10 +5,13 @@ module V3 = {
 
   @react.component
   let make =
-    (@warning("-16") ~a, @warning("-16") ~b, _) => {
-      Js.log("This function should be named 'TopLevel.react'")
-      ReactDOMRe.createDOMElementVariadic("div", [])
-    }
+    @warning("-16")
```

Comment:
This warning is added to `pexp_attributes` by JSX v3.

Reply:
Parsing and printing should not change anything for JSX right?

Reply:
Actually, this is printing change. The ast is not changed.

Reply:
I don’t think the printing changes of JSX output would be a problem unless the ast is same. What do you think?
I’ll check the js ouput for this case.

Reply:
JSX is never printed